x-pack/filebeat/input/gcs/types/types.go

```diff
@@ -0,0 +1,33 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// Shared types are defined here in this package to make structuring better
+package types
```

Comment:
Same question here. ISTM that most of the packages could live in the same place without any problem.